### DISCUSSION TITLE
Official Security Onion AMI now available in AWS Marketplace!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#3403] [Announcements]
Official Security Onion AMI now available in AWS Marketplace!
https://securityonion.net/aws/?ref=_ptnr_soc_dscs_210311

Documentation:
https://securityonion.net/docs/cloud-ami

If you have questions or problems, please start a new Discussion:
https://securityonion.net/discuss

![Screen Shot 2021-03-11 at 12 50 43 PM](https://user-images.githubusercontent.com/1659467/110836083-a36fee80-826d-11eb-86a7-210a3504cc46.png)

**Comment by yudansha10** [comment]
I'd like to take advantage of the AMI but is it really $0.15 / hour over and above the infrastructure cost?

**Reply by TOoSmOotH** [reply]
@yudansha10 You are correct it is $0.15 an hour for the AMI.
